charts/steadybit-extension-kubernetes/templates/deployment.yaml

```diff
@@ -143,6 +143,10 @@ spec:
             {{- end }}
             - name: STEADYBIT_EXTENSION_DISCOVERY_MAX_POD_COUNT
               value: "{{ .Values.discovery.maxPodCount }}"
+            {{- if .Values.discovery.disableArgoRollouts }}
+            - name: STEADYBIT_EXTENSION_DISCOVERY_DISABLE_ARGO_ROLLOUTS
```

Comment:
The env var is `STEADYBIT_EXTENSION_DISCOVERY_DISABLED_ARGO_ROLLOUTS` (missing `D` in disableD)